renderer-macos/lib/stubs/BriskTextView.c

```diff
@@ -52,12 +63,17 @@ BriskTextView *ml_BriskTextView_make() {
   return txt;
 }
 
+void ml_BriskTextView_setTextContainerSize(BriskTextView *txt, double width, double height) {
+  txt.textContainer.containerSize = NSMakeSize(width, height);
+  [txt.layoutManager ensureLayoutForTextContainer:txt.textContainer];
+}
+
 double ml_BriskTextView_getTextWidth(BriskTextView *txt) {
-  return [txt.layoutManager usedRectForTextContainer:txt.textContainer].size.width;
+  return [txt.layoutManager usedRectForTextContainer:txt.textContainer].size.width + (txt.textContainerInset.width * 2);
```

Comment:
This assumes you have an equal width/height padding, but somewhere you were trying to set arbitrary padding and iirc it wasn't working out for you.

Reply:
True, this will be solved in #74.